androidsdk/src/main/java/com/wootric/androidsdk/Wootric.java

```diff
@@ -83,85 +118,216 @@ public void setEndUserEmail(String email) {
      */
     public void setEndUserCreatedAt(long createdAt) {
 
-        if (createdAt > 9999999999L){
-            Log.v(TAG, "WARNING: The created date exceeds the maximum 10 characters allowed. " +
+        if (createdAt > 9999999999L) {
```

Comment:
Hmm should not you rather convert it to a date and make sure that it's valid and is not in future?